typescript-demo/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
+
+## Getting Started
+
```

Comment:
I know this is the default readme, can we add in the npm install step and then the npm lint commend as well to show a user how to lint?

Reply:
I had realised the default readme didn't include that. Pretty odd since the lint comes with it. I'll add that in

Reply:
done